src/style.css

```diff
@@ -1 +1,75 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, serif;
+  font-weight: 300;
+}
+
+.form {
```

Comment:
The .form class has a top and bottom margin of 20px. There should only be a top margin of 20px for the big search bar.